client/src/lib/stores/useRoomStore.ts

```diff
@@ -18,7 +18,10 @@ export type useRoomStoreResponse = {
     stream?: MediaStream,
     activeCamera: boolean,
     activeMicrophone: boolean,
+    usersInRoom: number,
```

Comment:
this `useRoomStoreResponse` is completely redundant

Reply:
I need this because i listen "ws.on('get-users)" and next I set count of users in store state